---
Date Generated: April 19, 2024
Transcription Model: whisper medium 20231117
Length: 5367s
Video Keywords: []
Video Views: 703
Video Rating: None
---

# BI 180 Panel Discussion: Long-term Memory Encoding and Connectome Decoding
**Brain Inspired:** [December 11, 2023](https://www.youtube.com/watch?v=2lo6iblgit8)
*  This is Brain Inspired.
*  Hello everyone, my name is Paul and welcome to another special panel discussion episode.
*  I was recently invited to moderate a panel amongst six people at the annual Aspirational
*  Neuroscience Meetup.
*  Aspirational Neuroscience is a nonprofit community run by Kenneth Hayworth.
*  So Ken has been on the podcast before on episode 103 and you'll be hearing from him shortly
*  discuss with me what the panel was about, what the meetup was about and so on.
*  Because there were six panelists from a broad range of backgrounds and interests and approaches,
*  I will let them introduce themselves to you after my chat with Ken.
*  But the goal in general was to discuss how current and developing neuroscience technologies
*  might be used to decode a non-trivial memory from a static connectome.
*  It's a mouthful, I know, but the discussion was about what the obstacles are, how to surmount
*  those obstacles and just in general what approaches should be taken and so on.
*  Just a heads up, if you're watching, there isn't video of the event, just audio.
*  And because we're all sharing microphones and they're being passed around, you'll hear
*  some microphone type noises along the way.
*  But I did my best to optimize the audio quality and it turned out mostly quite listenable,
*  I believe.
*  I do link in the show notes to the information for all the panelists and also aspirational
*  neuroscience and Ken.
*  Alright, so here is Ken, followed by the panel discussion.
*  I hope you enjoy it.
*  Long-term memory encoding and connectome decoding.
*  So listeners are about to hear the panel that I moderated with six guests.
*  And I thought that as the aspirational neuroscience person, I thought it would be good to maybe
*  just have a quick introductory conversation with you, Ken, and talk about the background
*  of how the panel got formed, whether and how it's going to continue into the future, and
*  what aspirational neuroscience is in general.
*  So again, just thank you for inviting me to moderate the panel.
*  And I enjoyed the discussion.
*  It was six people on a panel.
*  It was kind of a tough slaying in terms of just ensuring that everyone had their time.
*  But they all took a good amount of time, I thought.
*  So anyway, so thanks for having me do the panel.
*  Thank you.
*  So what is aspirational neuroscience?
*  What is this panel all about?
*  Why did we even have this panel?
*  Yeah.
*  So aspirational neuroscience is an outreach project, essentially.
*  The main goal is to bring neuroscientists together to talk about a particular possibility
*  that is presenting itself in the near future.
*  So I'm involved in connectomics.
*  I usually hang around with people that are involved in electron microscopy connectomics.
*  There has been just this incredible revolution from the point of view of being able to map
*  relatively larger pieces of brains than we ever have in the past at the synaptic level.
*  For example, over the last few years, there have been an entire fruit fly brain, actually
*  two entire fruit fly brains that have been mapped at the synaptic level.
*  And the Drosophila community are now pouring over that very detailed synaptic level connectivity
*  map and really tying that in with, for example, the attractor dynamics and the central complex
*  that allows the fly to know its orientation when it's walking relative to other space
*  and decoding the visual system of the fruit fly, et cetera, et cetera.
*  This connectomics revolution is really, it's not just fly.
*  There's a cubic millimeter of mouse cortex, visual cortex that the Allen Institute has
*  painstakingly imaged and is again kind of poised to revolutionize our understanding
*  of the mouse visual cortex.
*  One of the panelists, Anton Arkhopayev, is working with that team of people that generated
*  that connectome at the Allen Institute to ground his simulations of the visual cortex.
*  And this is only getting started.
*  So there is now the NIH just funded two groups, one from Harvard and Princeton collaboration
*  to do 10 cubic millimeters of the hippocampal areas.
*  Allen Institute.
*  Is that in mouse again?
*  Sorry.
*  That's in mouse again, yes.
*  Another Allen Institute has been funded to do 10 cubic millimeter, connectome 10 cubic
*  millimeters of the cortical basal ganglia thalamic loop.
*  These are incredibly interesting projects and they're getting to the volumes that neuroscientists
*  have dreamed about having this information.
*  And I should say that there is yet another revolution that's about to take over everything.
*  Electron microscopy is the reigning king right now for mapping connectivity at the synaptic
*  level, but expansion microscopy out of several labs now using pan staining of proteins has
*  virtually demonstrated that you can do connectomics using light microscopy.
*  And that has two advantages.
*  It has an advantage of being potentially faster and cheaper, which means that instead of doing
*  10 cubic millimeters, maybe you could scale this up to a whole mouse brain.
*  Maybe you could do multiple animals.
*  A lot of people in the panel was saying that doing multiple animals and having it
*  in line with a whole bunch of experiments would be the key to being able to really
*  decode these connectomes.
*  But the other thing is that this expansion microscopy approach means that all of the,
*  and I really mean all, all of the key biomolecules that people in the panel,
*  that we're going to hear, that people in the panel were saying, well, gee, I don't know
*  if you could decode things without having ion channels.
*  I don't know if you could decode things without having more detailed information about the
*  synapses.
*  This is actually on the horizon.
*  There are papers out there that have given connectomics level information using expansion
*  microscopy and then gone ahead and done antibody staining for the presynaptic and postsynaptic
*  proteins.
*  So that sets the stage.
*  So why are we having this aspirational neuroscience meetup at SFN?
*  Why are we bringing these people together?
*  Well, the idea is that these technologies are poised to map the nervous system in a
*  way that has never been essentially dreamt of.
*  The key question is, can we use that to test our theories of how learning and memory occur
*  at an individual level, not just to say what the general animal does, but to say what the
*  specific animal.
*  Can we read the algorithm?
*  Can we read memories off of these connectomes?
*  The community has been very divided over this.
*  Some will say, of course, that's what the textbooks say.
*  We need a lot of information.
*  We need a lot of more theories.
*  But of course, others will say, no, this is too complex.
*  This aspirational neuroscience outreach is really to bring those people that are doing
*  the connectomics together with the researchers that have the information that would potentially
*  be needed to decode the connectome.
*  People like Thomas Ryan, who's a leader in the in the memory engram literature, people
*  like Anton Arkhpaev, who is doing really detailed biophysical simulations of neurons.
*  It's to bring those people together.
*  We don't need to really detail how the conversation went.
*  Do you have any reflections on how you felt the panel went and how it might presumably
*  you're going to do it again next year?
*  It's supposed to be an annual thing now.
*  Do you have a clearer direction of how you might want it to go in the future, thinking
*  about how it went this past time?
*  If our donor base remains where it is right now, we will be able to have this conference
*  every year at SFN to bring people together.
*  I should also say that we have something that is, I think it brings everything together.
*  It focuses, it's designed to focus our thoughts on what is necessary.
*  And that's the idea we're putting forward.
*  The aspirational neuroscience outreach project is putting forward a hundred thousand dollar
*  prize for the first decoding of a non-trivial memory or learned function from a static connectome.
*  Do I know what non-trivial means in this category?
*  No, I don't.
*  This was one of the things that the panel is discussing.
*  But I feel like this is the thing that could focus people's minds.
*  As part of that, of course, whatever non-trivial means, it hasn't been done yet.
*  Almost by definition, it has not been done yet.
*  But if that is the goal, we want to focus on all of the pieces that are the sub milestones,
*  if you will, that are being achieved on the way to that goal.
*  Every year, until that goal is reached, every year the hope is to give four
*  aspirational neurosciences, neuroscience awards to different research groups that have really made a
*  significant contribution toward that overall goal.
*  For example, the four awardees this year, they all did fantastic research.
*  Let me just give one as an example.
*  There was actually two papers out of Tony Zadar's lab that demonstrated
*  the decoding of a trivial learned task.
*  They could decode from essentially a measure of the connectivity in a particular part of the brain
*  whether each mouse was trained to lick left to a high tone or lick left to a low tone.
*  And they could decode that training based upon looking at the synaptic connectivity
*  from the auditory cortex to the striatum and saying if these synapses are stronger over here,
*  then this individual animal learned one of those tasks.
*  And if it's stronger over here, then the individual animal learned the opposite task.
*  And so if we're listening to this panel discussion, there's this idea of, well,
*  is there any possibility that we can decode things?
*  The idea behind these prizes is to give people a very clear example of where we are today
*  in our theories and our attempts to decode trivial memories to kind of see where it might be leading
*  to decode a non-trivial memory.
*  Yeah, very good.
*  Yeah, there was a lot of talk about what is trivial and what is non-trivial.
*  Because there were different organisms being
*  represented in some of the different researchers.
*  There was a lot of talk about what might be trivial or non-trivial in various different
*  organisms and what might count.
*  And so I found it to be an interesting and fruitful discussion.
*  I know the discussion continued a little bit after the panel as well.
*  So yeah, anyway, I think that serves as a pretty good introduction to what people are going to hear.
*  Any final words or thoughts before to guide people into the discussion?
*  Well, I will say that I think this is just the beginning of a discussion.
*  So as you listen to the panel discussion, how I was listening to it, I was thinking,
*  OK, there's a lot of people that are talking about a lot of different things.
*  And so this has not coalesced.
*  But that's exactly what we expect.
*  And the idea is to put forward this challenge prize to really get this kind of idea of what
*  to really get this conversation going and to get it eventually to be at such a detailed level
*  that we're not talking about, well, gee, what does the memory really mean?
*  Or could this only be done in sea oligons and not in a fruit fly or something like that?
*  But to get down to particular experiments that are either going to work and tell us something
*  or they're not going to work and tell us something.
*  Very good.
*  Well, thank you for spending the time to help me introduce this.
*  We'll catch you soon, Ken.
*  Thank you.
*  We have a lot of panelists.
*  So I'm going to ask each of you to briefly introduce yourselves and why you think you
*  may be here, what you're studying, what your area of expertise is, because there's a lot of
*  different expertise here.
*  And then just in light of what Ken was saying, maybe what you think would be a convincing
*  non-trivial memory demonstration that would satisfy aspirational neuroscience.
*  Anton, do you want to start?
*  All right.
*  Hi, everyone.
*  So I'm Anton Erhippov.
*  I'm an investigator at the Allen Institute.
*  So what I do is modeling, biorealistic modeling of brain circuits.
*  I guess that's why I'm here, to have some modeling perspective on this question.
*  I'll say something in terms of what could be an interesting non-trivial memory that
*  could be reconstructed.
*  There might be better ideas.
*  There are probably a lot of different possibilities.
*  A lot of different possibilities.
*  But I would suggest try to decode images in the visual system.
*  So we study mouse visual cortex, for example, at the Allen Institute, among other things.
*  Can we try to decode images that a mouse is familiar with from the connectome?
*  And what percentage should the decoding algorithm get right for it to be considered a success?
*  Above chance?
*  That would be a good start.
*  Hello, everyone.
*  I'm Ji Hao Zeng.
*  I'm a postdoc in Princeton University.
*  I think I'm here because I've been working on high throughput
*  TM imaging for acquiring the kind of data set for connectomics for many years.
*  My PhD, I image a whole fly brain.
*  It sounds small, but I think it's the biggest at the time.
*  My postdoc, I continue to increase the imaging efficiency of
*  TM, transmission electron microscope, and also acquiring data set for hippocampus.
*  So that's brain regions that closely related to this topic.
*  For decoding a memory, I think the first minimal that it has to reach
*  is what I would call a beyond one-to-one correspondence between physiological changes
*  and the behavior outcome.
*  I wanted to draw contrast to some of the earlier very successful studies.
*  You can go all the way to Eric Kandel.
*  Gil Wejoo reflexes.
*  Everything is worked out.
*  It's amazing.
*  You have the molecule, the exact sicknesses, and you have the reflexes,
*  the habituation's behavior.
*  Everything is worked out.
*  So that's an example.
*  And the other example is, for example, fly T-Maze and fear conditioning.
*  But I think all of these is great success.
*  But one thing common about this is that there's a one-to-one correspondence between
*  the synaptic changes and what we call memory, which is the behavior outcome.
*  You have a fear conditioning.
*  Your synapses are strengthened.
*  You reduce the strengthening of the synapse.
*  You have less fear response.
*  And I think we need to do, I think we need to make the next step forward.
*  I mean, I guess the example I would say is that I come into this room,
*  my memory immediately formed with the discussion topic, the people I have known that are sitting here,
*  the context in which it was happening.
*  It just seems like you can't explain that with just a synaptic potentiation and reduction.
*  So I mean, I guess it comes down to it's a representational issue,
*  is in how the information is encoded in the brain in a systematic manner.
*  And I think the next step forward is to break away from the one-to-one correspondence.
*  And you can decode more than a binary fashion.
*  So, sorry, I got a little bit too long.
*  No, that's okay.
*  So following that, Conrad, what would be nontrivial in C. elegans?
*  So I guess I'm here because I'm really interested in simulating a whole C. elegans.
*  And I think that that's possible.
*  When you, if you ask me what's nontrivial, say, learning about aspects of foraging feels nontrivial.
*  But I also want to surface something that would feel trivial to me.
*  Like I can for human beings decode their memories by looking at the muscles that they have.
*  Because the muscles that they have tell me what activities they're typically doing.
*  I can distinguish a swimmer from a dancer just based on how the body is made.
*  And I think we would all be not happy with that.
*  So for me, it's a bit of a continuum.
*  Like one way of decoding is if you give me a simulation, I'll be happy.
*  Sure, that's decoding.
*  And if you give me like just the muscles, probably I'm not happy.
*  And I don't know like how to think about the space in between.
*  So that's why I'm so happy about the discussion we'll be having here.
*  Well, can you introduce yourself as well?
*  Well, I'm Conrad Carding from UPenn and I'm quite interested in causality.
*  Tomas, you are our resident in-gram expert.
*  I can lead off by saying that.
*  Thanks for making it, by the way.
*  So yeah, who are you and what will be nontrivial in your world?
*  Good evening.
*  I'm Tomas Ryan.
*  I'm a behavioral neuroscientist at Trinity College Dublin.
*  I believe I'm here because I work on what we call memory and gram ensembles.
*  I participated in this discussion at SFN 2019 before the end of the world.
*  And I think this is a fantastic space because it really allowed us to have a conversation,
*  to think outside the box and to be critical about many of the assumptions that we work with
*  pragmatically in neuroscience.
*  We recently published a paper, myself and my student Fiona Sullivan,
*  entitled If N-grams are the answer, what is the question?
*  And we were trying to assess our assumptions of what we think about when we think about
*  memory N-grams.
*  And I'm interested in N-grams as ensembles that may be embedding information in a wider
*  connectome.
*  And the challenge that we were really putting out there was that what we're currently
*  intervening with or manipulating as memory N-grams may not actually be the information itself,
*  but maybe the vehicles of the information, the biological vehicles of the information.
*  And the challenge really is to understand what aspects of these vehicles are carrying
*  the information so that we can then design experiments that would allow us to decipher
*  what that information is.
*  So to answer the question, what kind of experiment do I think would be appropriate?
*  I think we need to take inspiration from the only successful empirical science of information
*  that we have, which is genetics.
*  And the way we figured out what was carrying core genetic information was to transform
*  the putative substrate from one bacterium to another, showing eventually that it was
*  DNA that was carrying the information as a vehicle and not the protein.
*  And then we knew where to look.
*  We knew how to then chase down what the information might be.
*  I don't think we should be swapping N-grams or ensembles between different organisms
*  to find the substrate in this case.
*  My own personal view is that we should look at the relationship between memory and instinct
*  and how that happens across evolutionary time with the hope of finding the minimal
*  necessary components of what a latent informational structure might look like in the brain.
*  So my name is Dong Song.
*  I'm a professor at the University of Southern California.
*  So the reason I'm here, now I don't do anatomy, right?
*  But I do a lot of the E-phase in behaving animals.
*  And also my group, we do a lot of modeling, modeling of the hippocampus
*  with different approach.
*  One is the biological approach.
*  We try to build full-scale, very realistic model of the hippocampus.
*  And at the same time, we also build like input-output machine learning kind of model
*  of the hippocampus in the context of developing hippocampal memory processes
*  for enhancing or restoring memory functions.
*  So in the context of hippocampus, what's a non-trivial task?
*  So first, I must confess, I have been working on trivial tasks for very, very long, right?
*  For example, the binary discrimination task in terms of decoding, in terms of retrieving.
*  Now, we literally train the animal to remember left location versus right location
*  and to try to use the model to predict the activity, the representation for each location.
*  And then try to use the prosthetics to restore that memory.
*  So we have been doing that for a very long time.
*  So I don't think it's really trivial, right?
*  Still a good proof of principle.
*  But in the end, we also acknowledge that hippocampal function is not for binary discrimination,
*  right?
*  It's really for the formation of episodic memory.
*  So for me, in the hippocampus, a non-trivial task should include some kind of episodic memory.
*  There should be location involved.
*  There should be time.
*  There should be objects.
*  There should be the sequence.
*  So it's not just a binary discrimination.
*  It's a memory or memory trace of the episodic of events.
*  Thanks, and before you start, I'll just say, when you finish, we're going to have a broad
*  discussion.
*  And I don't need to speak at all if you guys want to talk amongst yourselves.
*  That'd be wonderful.
*  But I'll lead off with a question.
*  So go ahead, Srini.
*  Yeah.
*  Excuse me.
*  I'm Srini Turaga.
*  I'm a group leader at Janelia Research Campus.
*  I guess I'm here because I've been working on modeling connectomes.
*  Before that, I worked on machine learning methods for mapping connectomes.
*  Fun fact, deep learning algorithms were originally invented in their new renaissance
*  and applied to connectomics.
*  I'll one-up Conrad here and say that I think we can get to simulating an entire fruit fly.
*  Now that through Zeeha's work and Ken's work, we'll have maps of entire connectomes, the
*  entire nervous system for the fruit fly.
*  We're working on models of, so far we've done the fruit fly visual system.
*  We've also built a body model because you've got to embody the body.
*  You've got to embody these simulations.
*  So I think we have the tools we can do this.
*  And then to answer the question, I mean, I'm trying to think about this from the perspective
*  of scientists looking from the outside at a simulation or an animal.
*  The first thing that we strive for when Conrad and I build these models is to get the average
*  behavior of an animal right.
*  And we're kind of, that's where we're working right now.
*  But what would be a memory?
*  A memory would be, you know, some mechanistic basis for the individual response of an individual
*  animal.
*  So individuality, but in a way that's experienced.
*  So if we can kind of, I think, use the ideas that you guys are discussing, but phrase it
*  in the form of, you know, if I wanted to read it, then it has to be some measurement that
*  I make, which constrains something in the model, which then allows me to predict the
*  behavior of the animal at the end of the day in a way that's, you know, reflects its individual
*  experience.
*  Okay.
*  Thanks, everyone.
*  That didn't take the whole panel time to introduce everyone, so that's good.
*  So I guess I'll just remind everyone that the central focus of the panel discussion
*  is whether we can decode a memory from a static connectome.
*  And what I want to ask everyone, and you can just jump in when you want to, but the question is,
*  what in your own research and your own expertise would be holding you back currently,
*  would be the obstacle or an obstacle or a challenge, would be holding you back from
*  doing that?
*  You look like you're going to jump in.
*  I'd love to jump in here.
*  So from my perspective, causality is a big problem.
*  And I think we heard that earlier in your point as well, Thomas.
*  Which is, if we want to interpret something, we want to interpret it in terms of what it does,
*  not just it being a correlate of something.
*  And this, like, in terms of what it does is incredibly difficult because neuroscience,
*  by and large, outside of connectomes and perturbation studies, doesn't really tell
*  us much about the flow of causal information in it.
*  Does anyone want to respond to that?
*  I mean, I'll say, if you want to get at it from the level of a connectome, then we need
*  correlated measurements of connectomes, not just one connectome, but now multiple connectomes,
*  to see what differences there are in the connectomes of two different animals with two
*  different experiences.
*  And we're still at the stage of mapping one connectome.
*  We're now going to have a male fruit fly and a female fruit fly.
*  But as we make this faster and faster, maybe we can have two different fruit flies with
*  two different experiences.
*  We'll probably need more than just two, but that's what we need.
*  But eventually, the richness of our memories is a very individual thing.
*  So we would need seven billion connectomes, perhaps.
*  Or is that, or am I overshooting?
*  I mean, that's where models, I think, should be able to help.
*  I guess I can help by saying that the goal is that hopefully you have 10s or 20, and you have
*  some kind of statistically extracted understanding of the principle behind it.
*  So you could say something about the seven million animal.
*  So I was going to respond to the answer, but I'm going to essentially say the thing that
*  Serena was saying as a sort of EM image specialist.
*  I think we need to increase the efficiency and the scales of acquiring connectome.
*  And the first step is acquiring data.
*  And I guess there's subsequent steps about that.
*  There's reconstruction and then there's proofreading.
*  Ken was pointing out this big grant in the next five or 10 years.
*  And I think the goal is exactly to increase the...
*  People have realized that we need to be able to image a whole mouse brand and maybe image
*  many mouse brands, and we need to image many fly brands.
*  And we need to do experiment with the fly and image that fly brands in EM.
*  And these grants are to advance our technology such that we could do it at will.
*  And I think that's what I would think them in.
*  What we mean by decode memory, right?
*  It's a decode specific memory from a specific individual.
*  Or we also need to decode individuality of a specific subject, right?
*  To me, it's a totally different question.
*  So for the first possibility, I think it's much easier.
*  I think it's much easier to decode a particular memory for a particular subject.
*  But can you say that memory belongs to this individual person?
*  Right?
*  I don't want to go to the question of consciousness, right?
*  But obviously it's related.
*  So we really need to define what do we mean by decode memory.
*  I'm going to be slightly obtuse.
*  And I remember once Noam Chomsky remarking that filming all of the traffic in the world
*  and then quantifying it and modeling based on it would not be the most efficient way
*  of deriving Newton's laws of motion.
*  And similarly, I think that with the conectomics approach, which I completely endorse in itself,
*  if we take genomics by analogy, we didn't start sequencing whole genomes until the 90s.
*  It became high throughput around 2000.
*  That's not how we worked out the genetic code.
*  And it wouldn't be the best way, I think, of working out the genetic code.
*  I think the same is true for neuroscience and memories.
*  And if I ask myself, what is the thing that's getting in the way for us to make progress on that?
*  I don't think it's the tools.
*  Although the tools are always improving, we want better time windows of capture for memory and
*  grams, we want better tools for manipulating, damaging or reactivating them and so on.
*  But we're going to need a coherent and testable theory for what the informational structures are
*  once we have confidence of what the vehicle is.
*  And it's not going to come from decoding an entire conectome.
*  It's going to come from some kind of ideas about what are the topological or topographic
*  sub-motifs that lie within the conectome.
*  We don't just want to know what is necessary for learning or what is necessary for recall.
*  We want to know what the engram does or what we think the engram, what is it doing
*  for the organism.
*  And it seems to me that we should be doing that ultimately in C. elegans.
*  If we really are or similarly simple organisms, we should be able, if you don't understand a mouse
*  until you can build a mouse, we don't understand an engram until we can build an engram and surely
*  we'll want to be doing that in very simple organisms.
*  Why shield?
*  What is it about shield?
*  Simplicity.
*  302 neurons.
*  Because you have the conectome.
*  Well, it makes the conectome more easy to manage, but I don't think it needs to be about the whole
*  conectome.
*  You want to identify particular memories that have particular informational content
*  in a subset of the conectome.
*  Maybe you don't need to go as low as C. elegans.
*  Maybe you don't like all the neuropeptides.
*  So there's another animal one may want to use.
*  But we're always going to be limited when we're with mice and rats, I think.
*  So Sweeney, I want to push you a little bit.
*  I think you feel that C. elegans and fly are kind of almost at the same level.
*  And just don't think I buy this at this point of time.
*  And here's why not.
*  So yes, your work and a few other people are working on it show that you can do better
*  chance at saying things about neurons in the fly by looking at the conectome.
*  But in reality, the conectome just shows a structure and there's things happening.
*  There's lots of molecules.
*  We don't know what they are.
*  It's clear that there's some things that are missing.
*  But if you make statements like, oh, yeah, we have the conectome on the fly.
*  So we're at the same level.
*  Like the cool thing about C. elegans is that we can do the perturbation so that we can find out
*  what the functions are that neurons compute based on its inputs.
*  And I don't see us being able to do that on the fly anytime soon.
*  Okay, so in C. elegans, you can and real life has been doing that for a bit.
*  You can stimulate every individual neuron and there's tech development happening in
*  multiple labs.
*  I'm currently collaborating with that is aimed at basically being able to stimulate
*  subsets of the neurons.
*  And now you can say the output of each neuron if we follow like neuron doctrine is a function
*  of the inputs.
*  And arguably, we don't know how complicated that function is.
*  At some level, if the function is very simple, then we can get very far with the approach
*  of just looking at the structural conectome.
*  If that function is very complicated with time scales and nonlinearities and all kinds
*  of channel effects, then we need much more, at least in the structural conectome.
*  Maybe if we have the molecules, we can get there.
*  We can do stimulation of the flies, but the power analysis suggests that it gets very
*  difficult as you increase the number of inputs that each neuron has.
*  So C. elegans, it's not only that it has only 300 neurons, it's also that it looks like it
*  only has about like order 30 inputs per neuron.
*  And that means that they live in a much smaller space and that means that it's much more
*  realistic to more currently figure out what the nonlinear aspects of the functions that
*  they compute.
*  I want to push back a little bit more on you.
*  I think if you can agree that the fly does something more interesting that we wanted
*  to understand beyond the C. elegans and a conectome is a good starting point.
*  I mean, if you, if it is a possibility to perturb and recall in every single, that's
*  great in the fly.
*  That's, that's great.
*  But I think conectome is a starting point that raise the water, you know, raise all
*  boats for everyone for all technology to, to, to move to the next step.
*  The question is, what does that mean?
*  No, it raises the boats.
*  Can we, will we be able to predict rich behaviors of the fly based on the conectome of which
*  we only have structure?
*  I think you don't know until you can do it.
*  This, this, this, this, this, right.
*  But so what you say is we won't need C. elegans because the fly project will just work out.
*  No, no, no, no, no, no.
*  I think C. elegans is equally important for its set of behavior that it does.
*  But I think, you know, fly is a different organism that is equally important.
*  If somebody is interested in, in, in studying the fly, which I believe, you know, a large
*  fractions of the neuroscience community are then the conectome is useful.
*  Flights are wonderful.
*  Don't get me wrong.
*  I guess the question would be, you know, is any C. elegans memory non-trivial?
*  Right.
*  So C. elegans has a remarkably rich behavior repertoire and C. elegans can learn things.
*  For example, it can learn temple rhythms.
*  It can learn a circadian rhythm despite the fact that by default it doesn't have that.
*  It can learn aspects about food sources in its environments.
*  I would call that non-trivial.
*  But the question is kind of where does non-trivial start and trivial end?
*  So I want to mention something slightly different, but related to this discussion.
*  First of all, I think we need 8 billion conectomes.
*  But maybe for a different purpose than what we are discussing here in the future.
*  All right.
*  So like I said in the beginning, I think a good starting point would be to try to decipher
*  some kind of perceptual memory, not necessarily from a full brain conectome,
*  but it could be just, you know, the correlates of that memory in, let's say, a particular brain region.
*  And I want to suggest something that might be even simpler than that, perhaps on the way to that.
*  And I was inspired by Thomas' point about engrams there.
*  So in the last few years, many labs are pursuing this brain machine interfaces, right?
*  Where they use optical tools to let mouse train.
*  Let's say mouse, it can be done in other animals as well.
*  But let's say mouse trained to perform some kind of simple behavioral task
*  by learning to manipulate the activity of a few of its neurons, right?
*  Just a few neurons, let's say in the motor cortex,
*  that the mouse somehow learns to activate on demand leads to a reward.
*  Right? So can we decipher that from the local conectome of that piece of tissue?
*  And do it consistently for a few mice?
*  And that also goes back to the question, you know, is that transferable?
*  What is in common between those, you know, they're the same species,
*  but different individuals, what is in common there?
*  So this is not going to answer all the questions, obviously,
*  but I think that would be a good start.
*  What currently is holding you back from doing that in your own work?
*  Exactly. So what is holding us back?
*  I don't think anything is actually holding us back.
*  Something like that could be done now.
*  The existing data sets simply didn't do that experiment
*  before the electron microscopy was done there, right?
*  But it could be done, you know, mice can be trained that had been shown.
*  A piece of a visual or motor cortex can be taken out and processed.
*  You probably don't even need a full reconstruction of that conectome.
*  You could focus on some identified neurons.
*  So you just need to work faster. Is that the take home?
*  It's just go ahead and do that experiment.
*  If someone wants to do it, I think it's doable right now.
*  I want to add something to it.
*  When you talk about memory, keeping my memory, the behavior, right?
*  It's a very high level function.
*  A conectome is a very, very low, right?
*  No offense, but it's a very, very low level kind of mechanism.
*  There's definitely something missing in between, right?
*  I would argue that would be e-phase.
*  So before we move to the whole conectome, to the whole emulation of the behavior,
*  how about we simulate a single neuron, right?
*  Based on this 3D reconstruction of the single neuron,
*  see whether we can replicate the functional property in terms of e-phase, right?
*  In terms of the, you know, you receive a lot of inputs, how many, right?
*  What kind of output spice you generate?
*  I think that's a...
*  I like the spirit of this question.
*  And I think it also relates to what the level at which Conrad is sort of asking
*  about the C. elegans conectome as well.
*  Let's try to understand at the microscopic scale, how do the components work?
*  While I like that, I think our goal is, has always, at least my goal,
*  is to try to understand how this network as a whole gives rise to behavior.
*  And that's a network phenomenon.
*  And then, you know, there's, of course, the body itself and so on,
*  but this collective phenomenon, you can ask the question,
*  which of the details of that single neuron really matter to that?
*  And how important are the details of the single neuron
*  versus the connectivity of the network and the structure of that?
*  And my bet has been that it's the connectivity that matters more.
*  And, you know, we have some recent work that tries to investigate this,
*  but, you know, when you think about the kinds of,
*  you know, in machine learning, there's the architecture of the network,
*  and then there's the components, neuron properties that you use.
*  And what matters a lot is the architecture.
*  What matters a lot is how you combine them together in a network.
*  And I think we need to sort of, you know, go from the microscopic level
*  to the macroscopic network level and the behavior level.
*  And I don't think making more measurements at the microscopic level alone
*  are sufficient to transcend these scales.
*  And so a lot of our work has been about how do you bring in
*  constraints or measurements also at the macroscopic scale,
*  not just, you know, making more measurements at the microscopic scale?
*  How do you combine these measurements at different scales
*  into a single model and then use that?
*  And that may not give you the information about which neurotransmitter is being used,
*  which molecular mechanism is being used by that circuit to generate the behavior that it is,
*  but you may have effective parameters that can allow you to bridge these scales.
*  And to me, that's more important.
*  That's more important.
*  And this is kind of why I think, you know, it doesn't necessarily matter whether we can do
*  single-sendal perturbations in the fly connectome if we can measure lots of behavior.
*  And if we can measure lots of sort of, you know, network dynamics at some larger scale.
*  I have a sort of experiment that I think can support that.
*  And I would use, you know, the fly central complex as an example, right?
*  I mean, Ken mentioned about attractor dynamic in the hippocampus, but I think a simpler
*  attractor dynamic that we all know is the fly heading direction.
*  And this is, you know, work by Vivek Jayaraman, Gabby Memmon.
*  You know, you have the dynamic and we have the theory, right, for many, many years.
*  We have the connectome, but I think one more thing that need to make the bridge
*  from single neuron to that connectome is biophysics.
*  We feel like we haven't totally nailed out the mechanisms of the attractor dynamic
*  in central complex because we don't know what's actually contribute to the excitation
*  and what's contributed to the inhibition and how does excitation inhibition interplay to
*  generate the attractor and to maintain the attractor.
*  And I think more biophysics could, you know, sort of bridge this understanding from single neuron
*  to a network.
*  You know, you just know measure how much of the output input is contribute by each component.
*  And that could also totally nail down what's the modeling mechanism that's responsible for
*  this attractor dynamic in a real, you know, living system.
*  With the advance of machine learning, right, now people don't have to do very detailed,
*  right, biophysics modeling.
*  People can just get many, many samples of 3D reconstructed neurons and also get a lot of,
*  you know, e-face data from the neurons and let machine learning to figure out what the mapping is.
*  Maybe that can be a translator, right, translating single neuron to functional units.
*  But ultimately, if we want to do that, the central thing that we need to know is how
*  complex is that thing that we're trying to approximate with machine learning.
*  Now, that's what determines how much data we need.
*  And I think in the panel, we probably had a lot of disagreement on how complex different
*  animals are.
*  So I think humans are just mind-bogglingly complicated.
*  That just might be because I'm not very good at understanding them.
*  But still flies just seem like unbelievably complicated.
*  And even already a C. elegans, I'm not sure if I have any chance to understand it.
*  And then the question is kind of how many parameters do we need?
*  And if there's significantly more parameters, then we can do measurements.
*  Then kind of there will always be a set of models that are all compatible with what we have.
*  Now, that leads us a little bit away from decoding memories.
*  But it's still kind of if we talk about understanding how a system computes,
*  it leads us down that path.
*  But getting back to the decoding memory, let's say C. elegans.
*  So there's a constraint on the types of behaviors that they do perform.
*  And you mentioned a handful of them.
*  And does that give you some hope that you could, despite not being able to understand
*  C. elegans, whatever that means, that you could decode those 12, what you call non-trivial,
*  behaviors slash memories?
*  Yeah, I mean, like with C. elegans, and this is, I mean, like, please don't quote me on it.
*  But like, I'd imagine that kind of the number of dimensions that C. elegans can learn might be like
*  100 dimensions or something.
*  I don't know.
*  I mean, like, please, I might be many others of magnitude off.
*  But there's no doubt that humans can learn an absolutely mind-boggling number.
*  I mean, people can learn box-by-heart and things like that.
*  So in that sense, how hard it is to read out behaviors depends on how many parameters kind
*  of go into those behaviors.
*  In this forum, we all seem to agree that the connectome is the implementational substrate
*  that we want to be looking at.
*  There are people outside of this room who would argue it may be other levels of biology.
*  But within the connectome world, on the one hand, we're studying synaptic properties and what
*  is necessary for the plasticity that might underlie a specific engram.
*  And then we have this wide map of different extents to which we can model the connectome.
*  And the problem, it seems to me, is that we don't have any good way of defining
*  the questions that we want to answer in between a whole connectome and what's happening
*  at a synaptic level.
*  We don't really know whether whatever the engram is, whether it's truly distributed
*  and fragmented or whether it exists in modular components in different parts of the connectome.
*  More than that, we don't really know if whatever components of the engram we want to be dealing
*  with are intrinsically storing information somehow or contain information in a way that
*  is entirely relational and that only has any meaning with reference to the world from the
*  perspective of that organism.
*  And unless we start to get concrete about these different possibilities and how they map on
*  to different alternative possibilities, we won't be able to design testable experiments
*  to make sense of this.
*  And I worry that when we talk about mapping in vivo physiology, synaptic input, output,
*  at whatever level of complexity onto the connectome, what we're doing is we're just
*  studying biochemistry in parallel with the phenotypes of individual cells in order to
*  find how latent information is stored there.
*  But that didn't work again for genetics.
*  We needed to have more concrete ideas that were testable in a different way.
*  So just briefly to push back, you said we all believe that conic tom is like the correct
*  description.
*  I believe that if you don't also give me the molecules, you can learn very little about it.
*  What could we go without the molecules if we had an artificial connectome that could
*  behave the same way as a biological connectome but without those molecules?
*  Because the question is whether the molecules are simply there for operational activity
*  in the connectome plasticity and so on, but that the information is solely embedded at
*  the level of the network structure.
*  I absolutely believe that things like channel densities carry information and contain
*  significant parts of the memory.
*  Well, so in my opinion, if you want to simulate the brain or part of the brain faithfully,
*  you need more than the connectome.
*  You need all the electrophysiological properties of neurons, their connections, ultimately
*  how they are modulated by the mind-boggling complexity of neuropeptides and everything.
*  So in principle, we need to know all of that and the connectome is probably not enough.
*  It's possible that we might be able to learn associations between, say, from the connectome
*  or morphological reconstruction, I can tell what cell type is, what neuron, which cell
*  type it belongs to.
*  And if I characterize those cell types previously across all the types of conditions, I might
*  be able to use that in my simulation.
*  But just connectome by itself, I think we probably all agree is not enough.
*  However, for this particular question we are discussing here is what can we decode from
*  the static connectome that probably can be acquired right now.
*  Yeah, personally, I don't know.
*  Actually, that's an open question in my mind too.
*  I suspect that a few of the things like what we were discussing are possible to decode,
*  at least above chance.
*  But it's also quite possible that some of that information is not going to be able
*  to be resolved at the level that is not resolvable with the current technology, like phosphorylation
*  states of proteins or something.
*  Let me just interrupt and just remind the audience to ask questions at will whenever
*  you want to come up to the mic or raise your hand and I'll yell at you.
*  Yeah, I kind of want to ask a question.
*  I'd like to understand this idea that in the genome, we didn't really need to understand
*  and sequence the entire genome to understand how it works.
*  How do we bring that idea down to memories?
*  Because the way I think about it, a memory doesn't exist in the abstract.
*  There's some representation of the sensory input that's potentially unique from animal
*  to animal, individual to individual.
*  What a memory is the delta between where you were when you started and where you ended
*  up after that experience, similar to maybe like SNPs, single point mutations that might
*  be different between your genome and mine.
*  And so we needed some baseline to figure out where that difference is and then that
*  comparison helped us.
*  Now for connectomes, I mean, or for brains, if each individual has a different baseline,
*  then we need to figure out how that baseline works.
*  Or if we know how to map an input all the way to the memory module and compute from
*  the connectome of the rest of the brain what that sensory representation might look like,
*  then we may have some way for that individual mapping what that memory might look like of
*  a particular stimulus and try to decode that.
*  But I just want to get a feel for what I'm missing.
*  I'd love to learn.
*  So I would argue that we can learn a lot from the genetics analogy because we were able
*  to learn that there are modular structures in our genome, which are genes, which are
*  implemented in the vehicle of DNA.
*  But within that structure, within the chemical structure of DNA, what emerged in evolution
*  is an arbitrary informational code that was not determined by anything about the chemistry
*  or biology of that molecule.
*  The RNA sequence is directly complementary to the DNA sequence, the mRNA sequence.
*  So that has a physical mapping.
*  You go from DNA to RNA because of a structural complementarity.
*  But then something else happens because the way that protein amino acids are connected
*  to t-RNAs is based on an arbitrary mapping that is not structurally determined by anything
*  to do with the chemistry of the t-RNAs or the amino acids.
*  It is ubiquitous.
*  The chemical connectivity between an amino acid and a t-RNA is generic to all of the
*  combinations that the genetic code triplet has with the amino acids.
*  We just keep them consistent for the arbitrary reason that we need to be able to reproduce
*  with one another as organisms.
*  We need to have the same genetic code, but you could rewrite the genetic code any way
*  you want.
*  It's plug and play.
*  So at some point, we had this semiotic informational transformation where it stopped being just
*  about chemistry and structure and it started to become about a biology of information.
*  And it's not that all of genetic information is embedded in our genes.
*  They're not because, of course, we have developmental systems.
*  And what happens innately is a product of cells interacting with one another.
*  So we also have dispositional information in our genome.
*  But at a very basic level, we have these units of segments of DNA that are coding for protein
*  as latent informational structures that can be read by aliens from another planet even
*  when we're all extinct.
*  And it is objectifiable information based on a subjective code that evolved for specific
*  reasons in our evolutionary history.
*  Now, in order to take some of this reasoning to how we study the brain, and Michael Gazaniga
*  has made this point very nicely in his book, The Consciousness Instinct, we need to be able
*  to come up with ways of interpreting, in my opinion, subsets of connectomes in a way that
*  we can make predictions about what they are doing for behavior and then test those predictions
*  in functional experiments.
*  I completely agree with you.
*  The engram is the delta between your brain before an episode and your brain after the
*  episode.
*  The problem is there's a lot of delta going on in your brain because you're learning lots
*  of different things.
*  Your brain has to do a lot of work to stay alive.
*  You're aging, degenerating, and so on.
*  So the empirical challenge is separating out all of the irrelevant delta, all of the delta
*  for other memories, and all of the delta that's just homeostasis from the true changes that
*  are in fact laying down information that is the engram of interest.
*  I don't understand the RNA analogy.
*  In the case of RNA, the decoding is the same across all our cells.
*  In the case of neurons, the decoding is different for every single cell that we have.
*  Why does the RNA analogy apply?
*  The point about the RNA analogy is that we go from structural determinism to a degree
*  of semantic arbitrariness for how the code works.
*  There is nothing physically determined about the relationship of our 20 optional amino
*  acids with the triplet codes that code for them.
*  And something like that is going to be happening at many more levels in the brain by completely
*  different mechanisms.
*  We don't currently have any kind of a theory for how we would even go about doing that,
*  but it doesn't seem to me plausible that the way our engrams are expressed and interpreted
*  physiologically, not then behaviorally and cognitively, is a physically deterministic
*  thing.
*  At some point, we evolve internally restricted rules that became consistent in animals at
*  some point in our evolution.
*  We need a way of understanding that.
*  Okay, we have a couple questions.
*  Can you hold on one second, Randall?
*  There's a question right here, right?
*  So I actually wanted to raise two questions.
*  First one relates to the C elegans.
*  So how would you define a non-trivial memory while we know that, for example, a behavior
*  like circadian rhythm that you mentioned could be delayed with a very simple chemical reaction
*  like redox reactions, for example, has been shown before red blood cells that with very
*  simple reaction code could be done at the circuit level.
*  How would you define a non-trivial memory?
*  I can't define non-trivial very well.
*  I have the problem also artificial scotoma, which was like the big thing that was about
*  decoding relevant information from connectomes when I was a young PhD student.
*  I can't find a criterion that cleanly cuts between that and more interesting things.
*  So yes, I take back.
*  I don't want to strongly say it's non-trivial.
*  So the second question is related to the deep learning parallelism.
*  So the work from Anthony Zaito, for example, the genomic bottom and learned behavior.
*  So I wanted to hear your opinion about that.
*  So when you have, it could be that there are two different learning algorithms, one with
*  optimization for a circuit with a certain motor sequence and then shuffling between
*  these motor sequences and learning them.
*  How are you going to draw from a static connectome?
*  Yeah, I think that's a great question.
*  I think it does point out that we need to understand first what's the baseline behavior.
*  We have to correlate the connectome and the behavior to the genome and figure out which
*  things are genetically defined and which aspects are learned through experience.
*  And until we have lots of measurements, we won't be able to de-correlate them.
*  We just need more measurements.
*  Randall.
*  All right.
*  I've heard some really interesting debates between all of you about what is a non-trivial
*  memory and about having a connectome that isn't too complicated to deal with.
*  And I'd love to know this.
*  So here, it just says an idea.
*  So how wrong is this or how right is this?
*  How beneficial, how not beneficial?
*  If instead of C. elegans, fruit fly, mouse, let's say you take neurons cultured on top
*  of an electrode array where you can see everything through a microscope, you can record, stimulate.
*  And now you take, for example, your connectome data and you take the data from the network
*  and you're trying to deduce what are the things we see in it.
*  Either you could use the correlational approach to a non-trivial memory or the decoding circuit.
*  You can figure that out as well.
*  What could you get from that and what could you not get from that?
*  Maybe perhaps you can figure something out about what matters, what doesn't, a sort
*  of scale separation issue.
*  I would love to hear your opinion.
*  So I think that that's a great idea.
*  It would be very interesting and useful.
*  But ultimately, we want to be doing this with real brains.
*  I think the big question there is how well it translate to a real brain because I don't
*  think we know very well how in such culture or in organoids, the connectivity is organized.
*  It might be similar to real brains.
*  It might be vastly different.
*  Personally, I suspect that it's vastly different.
*  And also, you don't have a lot of the systems, neuromodulators, for example, that orchestrate
*  a lot of function, including memory, in the real brain.
*  And so, I mean, I think it would be a very interesting research question and a program.
*  And ultimately, comparing that with real brains would be fantastic.
*  In my opinion, we don't know currently whether it's really going to translate well.
*  To me, I think it's less important to have the cell in culture or in the body.
*  To me, I think the more important to find out whether that's a clear relationship
*  between the structure, 3D structure of the cell and its functional property.
*  So I don't think that will change a lot from culture to...
*  Now, even if it changes, as long as it's a single, one-to-one mapping, it solves the problem.
*  You can always translate structure into function.
*  But I really think that's a very necessary step, mainly because I'm doing EFACE.
*  I feel very uneasy about going directly from plankton to behavior.
*  But I want to give this to Randall.
*  If we can produce a small set of neurons, say three neurons interacting with one another,
*  at that size, we can probably identify how they interact with one another.
*  We could probably use a definition of learning, where you can say,
*  to which level does the connectome reflect the learning processes that have happened before?
*  Maybe the inputs we give to one neuron.
*  If we could do that, I would be very satisfied with it.
*  Personally, my belief is if you just look at the connectome, you will have much less than
*  if you also give me, maybe, say, the densities of various molecules.
*  But indeed, this is something that we could absolutely do on a small circuit.
*  And I, for one, would be very excited about this experiment.
*  Thank you.
*  Okay. I might be naive in saying this, but I think there's some overcomplication.
*  Could you take, for instance, C. elegans, which is like this simple organism,
*  and then look at circadian rhythm as a biological function, and then see how
*  the connectome changes over time for different light effects?
*  Could you get a memory of how C. elegans is performing in a specific light situation?
*  And then that would be a source of where you find an engram?
*  I'm not sure I understand.
*  We could have animals that have long circadian rhythm versus those that don't.
*  And we can look at the connectome and maybe it's different.
*  But where's the extra level of complexity that we can take out of it?
*  Well, I guess I'm thinking of the simplest situation.
*  To start to actually have a verified weight of saying, this is a memory that we're able to
*  derive, and it's without question related to a difference in a light condition, for instance.
*  I mean, obviously the more complex ideas are really good too, but to start small before
*  getting bigger, because obviously when you get to a higher level organism like a human,
*  the level of complexity is enormous.
*  So I'm not 100% sure I understand the question.
*  There's two pieces.
*  One is, can we establish that light makes the change in the connectome that we could see?
*  Versus the second one, does the change in the connectome that we see,
*  is that ultimately what gives rise to behavior that we see at the beginning?
*  The first part is much easier because we can randomize the light condition.
*  The second part is very, very complicated because it's a very nonlinear recurrent system.
*  So a comment on this one, just in general, C. elegans.
*  There was a paper in Nature a couple months ago, I think.
*  To my shame, the name of the author escapes me.
*  What's that?
*  Andrew Leifer.
*  Yeah, right, right.
*  Yes.
*  The author is Randy.
*  Thank you.
*  Perfect.
*  Yeah.
*  So what they showed is that not even thinking about behavior, but thinking about neural activity,
*  their experiments and modeling showed that connectome is not enough.
*  There was a lot of bulk transmission, right?
*  So non-synaptic communications between neurons that, according to that study,
*  really changes how the activity is organized very substantially.
*  And so, yeah, according to them, connectome is not enough.
*  To our earlier point, physiology, molecular diffusion and things like that are really
*  important.
*  I think it doesn't mean that memories cannot be decoded from C. elegans or whatever other
*  connectome.
*  It just means that a lot of the things may be much more complicated,
*  but maybe some things can be decoded.
*  I'm still thinking about how do we define decoding memory, right?
*  This fundamental problem.
*  So I think sometimes when we say that we mean, we literally mean we decode memory,
*  almost imply some kind of discriminative model, which is much, much easier.
*  But sometimes we mean something like a creator memory.
*  I think this order is more complicated than decoding memory.
*  So that question, can we decode memory from connectome?
*  Based on what Ken presented today, I think the answer is already yes.
*  Of course, those papers already indicated some of the simple memories can be reliably
*  decoded from the connectome.
*  But whether we can create in the real simulation or real animal,
*  that's a totally different problem.
*  So.
*  Ken, this better be good or I'm kicking you out.
*  Okay.
*  Okay.
*  That's fine.
*  So I want to call to question kind of this idea that every molecule is precious and
*  we don't know anything about neurons.
*  I mean, I've seen Anton's work and I believe you do biophysical modeling of whole neurons.
*  And so to put this in perspective, so there's this great paper.
*  That was looking at monkey brains and looking at the response of monkey brains.
*  Everybody's seen this figure where they can do 250 milliseconds.
*  They can respond to a complex visual stimuli.
*  That means they only have 10 milliseconds for each stage, essentially.
*  This means that it's electrophysiology.
*  This is not memory.
*  This is not gene expression or diffusion or anything like this.
*  It means it's electrophysiology of neurons.
*  It means dendritic integration, which might, obviously,
*  ion channel densities are incredibly important.
*  Do we know nothing about ion channel densities?
*  Because if you were to say we really have no idea about the ion channel densities of
*  particular cortical neurons, then how the heck do you ever get a visual neuron simulation
*  to give you responses?
*  If I look in the literature and I see people actually decoding trivial memories,
*  like the receptive field properties of a visual V1 neuron,
*  and they're not like, holy crap, it has nothing to do with the synapses.
*  It's like, yeah, it's being built up by the synapses.
*  So I want a concrete question of what are those specific ion channels
*  that we don't know that are varying in a way that are unpredictable?
*  And if we don't have that, then what are the experiments that we can do to get those?
*  Because I think that we can, especially with expansion microscopy.
*  All right, so I'll start.
*  So I think we know a lot about ion channels and their densities, although,
*  as far as I know, we don't know enough.
*  There is a lot that we don't know.
*  There is a lot of complexity for one thing, ion channels come in all kinds of
*  combinatorial assemblies.
*  What we know, it's very complex.
*  Also, what we know, for example, for modeling work, is that you can get the same answer
*  for the activity of a neuron given a stimulus with many possible combinations of parameters.
*  And honestly, that's how it works at this point.
*  We are not saying, well, there is this much of these ion channels, that much of that ion channel.
*  This is their distributions along dendrites, because we learned it for hundreds of them.
*  We assume like 10 different conductances with some properties that are roughly known,
*  and then we optimize them.
*  And we can get multiple solutions that can give you the same result.
*  So we get a working model that probably has some relationship to reality,
*  but it's not necessarily one-to-one relationship.
*  But all of that being said, I think I would totally agree with you.
*  We know that a lot happens at the synapses.
*  We know that paper from Kevin Martens' lab that you showed that got the prize.
*  It's great.
*  And things like that have been done before, but this is really sort of nailed it.
*  There is a relationship between how strongly these neurons are connected
*  and what we can learn from the connectome.
*  Not even full connectome, just partial electron microscopy volume.
*  So personally, I believe there is a lot that we should be able to decode.
*  But there are molecules, right?
*  There might be protein aggregates and assemblies,
*  or maybe even free-floating proteins with some kind of phosphorylation states
*  that determine modulation of that synapse, which may very well have an effect.
*  We just probably don't know yet.
*  Let me say a few things about channel density.
*  So the first one is traditionally we don't have the means at all
*  to see how they're distributed over a given cell.
*  Therefore, all the old biophysical modeling assumes that
*  channel densities are constant within a given cell type.
*  That assumption has never been properly supported by data.
*  So that's the first one.
*  We don't know how complex they are.
*  The second one is I want to make a normative argument.
*  So once neurons figure out how a synapse strength should change,
*  what's a channel?
*  It's just like a synapse that's always active.
*  So in that sense, we should strongly expect that
*  channel densities are actually something learned
*  and there's some evidence that they are.
*  And so in that sense, yes, I do believe that channel densities
*  carry significant information.
*  It's not that none of these things are important.
*  It's whether we can build simple, effective models of the network
*  without making all those measurements at the microscopic scale.
*  Can we get macroscopic behavior cheaper?
*  And we have.
*  I mean, this is not the first time people have built multi-scale models.
*  Chemistry is full of it.
*  There's quantum mechanical simulations.
*  You need them for certain kinds of things.
*  There's molecular mechanics models of just atoms.
*  You don't model the electrons.
*  You know, there's simple models of that.
*  There's models, effective field models of water for hydration.
*  There's all of this.
*  We have these models at different scales.
*  We don't need to measure everything at the finest scale.
*  Yeah.
*  Let me steelman and stormend that argument.
*  There is a scenario about the brain where interactions between
*  neurons basically stabilize things.
*  So if you get individual neurons wrong,
*  the fact that there's many of them that all interact will basically fix things.
*  There's another set of scenarios where any mistake you make
*  accumulates as it's recurrent in the system.
*  I don't think we know really where we are on that continuum at this time.
*  I agree with that.
*  But we can make measurements at the macroscopic scale
*  and constrain the macroscopic.
*  We don't just need to make more microscopic measurements.
*  Sure.
*  We can combine microscopic measurements with macroscopic measurements
*  to make effective models of single neurons.
*  But then the question is, if you get the individual neurons kind of wrong,
*  how much can you correct it by say having models of measurements,
*  models of behavior or measurements of behavior.
*  But at the same time, you could say if your models of individual neurons are too bad,
*  then why do we even need the neurons?
*  We can just look at behavior and fix everything on the behavior modeling level.
*  I totally agree with that actually.
*  For example, you can do your type of mapping at a functional imaging.
*  Functional imaging can also tell you a lot about your memory.
*  But that's not what we are trying to do here.
*  We still try to go to the connectome, go to the molecular level,
*  look at the computational principle,
*  try to not only decode but sort of reconstruct this machinery of the brain.
*  It's not just a classification.
*  It's almost like building a generative model of the brain.
*  In the spirit of having a little over 10 minutes left,
*  let's move on to another question.
*  Hi, I'm Justin.
*  I think that we can simulate the Drosophila connectome after decoding memories.
*  One thing that encourages me is that the biggest place we see
*  inter-individual differences in both the larva and the adult is in the mushroom body,
*  which is strongly associated with episodic memory.
*  Now that we have two adult connectomes,
*  we see the largest differences are in the memory center, the mushroom body.
*  However, there's a potentially confounding variable that homeostatic differences
*  can lead to large changes that aren't necessarily related to subjective differences in experience.
*  So I wanted to ask, how do we draw the line between what is a large homeostatic difference
*  caused by large rearing conditions and what is a specific inter-individual
*  or intra-individual difference over time that you would consider a memory?
*  I think that it's an articulation of the problem that those of us in the functional neuroscience
*  of memory have to either engage with or else we're just not making progress.
*  We're usually dealing at multiple levels anyway when we start perturbing a system,
*  and we have problems of causality that have already been alluded to by Conrad on that topic.
*  My view is that we're dancing around an elephant in the room that we haven't quite defined,
*  that there is an in-between level that we need to be dealing with where we may find that semantic
*  content. I don't mean that the in-between level is in vivo physiology and we simply need to map
*  that on to the connectome at any of these levels. What I mean is that at some point, an individual
*  animal is making a subjective use of a representation. The representation, I mean
*  the internal informationally rich version of a representation, I don't mean a correlation,
*  is drawing on whatever latent information you have at the relevant level of your connectome.
*  This is being operated on not in a deterministic way, behaviorally, which is why just simply mapping
*  conectomics onto behavior is going to lead us down, I think, the wrong kind of corridors.
*  Emerging from that information is a representation that an individual can use subjectively.
*  In neuroscience, we are used to looking at hypothetical or putative substrates of
*  representations in a third-person format. We need to be thinking more and more about
*  neuronal representations from the first-person perspective. There we have to consider multiple
*  realizability, that it's very different between individuals and it can be very different within
*  an individual at different moments. This point has been made by Romain Bret. It was made very
*  nicely at this Society for Neuroscience meeting by Andre Fenton in his special lecture. The point is
*  that looking for something that's always going to best predict the behavior is missing the point
*  because you can get there by multiple different ways depending on the state of the organism and
*  depending on the individual. Of course, we're going to be tied to particular substrates of
*  analysis, but unless we have a way of conceptualizing how that connectome is expressed
*  in a meaningful way as a versatile representation, we won't be able to explain how that informational
*  substrate in the connectome leads to adaptive behavior. We'll only be able to predict typical
*  behavioral outcomes, which is a very different thing. Another question? My question is, what
*  is the minimal memory unit that we're looking for and can we find that in a connectome or not?
*  Meaning, are we looking for an ensemble or sub-ensemble of an engram? Are we looking for
*  set of specific molecules? Are we looking for set of specific pattern connections? What exactly
*  are we looking for in a non-trivial memory to be able to decode it and can that be embedded
*  into a specific model? I think that what we've been looking for, and you guys can correct me,
*  is the behavioral output via experiment of the encoded memory, whatever that minimal unit is,
*  that's just above non-trivial, wherever non-trivial is. One extreme number would be single neuron,
*  in many cases, a single neuron can decode non-trivial information, like a location,
*  if the behavioral outcome is just a binary, you may get that information from a single neuron.
*  I mean, I would say that we have to make the next step forward to get away from binary and
*  one-to-one correspondence, and we need to have a minimal model. I think it's a system level
*  question. You need to have competition. An example I would say, perhaps in light of the study,
*  we all know play cell for a room, and if you can decode the location of the room,
*  representation of the room from a connect-home, a VR play cell, and from the connect-home,
*  that would be non-trivial. That seems not too far from the understanding we're having now.
*  One of my questions to you all was going to be, is decoding the right question, because
*  just because you can decode the location in the room from a given cell, you have to have known
*  that that location was visited before, and decoding it is kind of trivial. So something
*  non-trivial would be to decode which city someone visited in 1978 from that one cell.
*  I would just say that that's the next step questions after the first step. I would say that
*  this step I'm talking about is already non-trivial, and yours are equally non-trivial,
*  much more non-trivial, I would say. If we had a simulator, then it would be very easy. We could
*  ask the simulation what happened on that day, provided you still remember.
*  I was not born.
*  Okay, that solves it. But I wanted to push the organizers of this price a little bit.
*  Here's a very easy way to store a lot of information. I go into someone's retina,
*  I have a very bright laser, and I kill a bunch of neurons, and I can draw a whole picture of
*  things in the retina. Isn't that more non-trivial than anything we've been discussing the whole time,
*  and I have no doubt that I could decode which neurons I zapped with my laser from the connector.
*  So you don't mean non-trivial information, you mean like advancing science.
*  Okay, let's end up with one more question from the audience.
*  I'm Edmund Rolls. I work on the vertebrate hippocampus, and everything is totally different
*  there. What we need to know is the rules of operation of the system, rather than what happens
*  at a particular synapse. So the crucial things we need to know there are connectome related,
*  but there for example, the number of synapses that you have on a single neuron from recurrent
*  collaterals in CA3, because that sets exactly the memory capacity of the system. So that's a bit
*  missing, we need that. It turns out that we have diluted connectivity in CA3, and one advantage of
*  that we think is that then it decreases the probability of having two synapses, a pair of
*  synapses, or two synapses between a single pair of neurons. It turns out that if you have a theoretical
*  model of how attractor networks operate, that kills the memory capacity if you have them. So what I'd
*  like to encourage us to think about is things in a slightly more statistical sense, that we shouldn't
*  necessarily be worrying too much about an anatomical change term, a particular synapse in a
*  single organism. Pertubate brains work slightly differently to that. You've got very large numbers
*  of neurons, you're interested in the statistics of the thing, and you need to know things like how
*  many connections you have onto particular neurons. So it doesn't mean that the connection's not
*  important, it's crucial, but we understand how it's crucial in setting something like the memory
*  capacity through the theoretical physics approaches. So it's just a slightly different
*  way of looking at some of these questions. I really like that, and I think, you know, I was
*  thinking a lot about Justin's question too, which is, you know, let's say, how do you distinguish
*  between, you know, an animal just growing and preferring a particular thing just from birth
*  versus some experience causing it to like a particular substance? And one would be, we'd call
*  that a memory, the other we'd call innate, and it could just be an individual bias, that particular
*  individual just decided, you know, was just happened to prefer a particular thing. How do we
*  make that distinction? I mean, what matters is, in this case, A, the experience, but B,
*  the mechanism of forming a memory. You know, it does matter how we're making these memories and
*  how there's this correlation between experience and whatever changes there are in the anatomy or,
*  you know, other parts of the nervous system. So that means that the anatomy by itself is not
*  enough. You know, we need to know something about the formation process in order to distinguish
*  between the memory versus non-memory. And that also means it's going to be non-trivial to be able to
*  answer this question in the first place. We have to wrap it up so both of you make it rather quick.
*  I like Dr. Rieu's comment, right? But I think there's a small difference here. So what you
*  talk about is more like a model-driven statistical understanding to the connectome, right? Connectome,
*  in that sense, is raw data, right? But what you are trying to replicate is the memory function.
*  And I think what we are trying to talk about is memory, a particular memory. So even we can
*  replicate that memory function, that doesn't mean we can replicate a particular memory.
*  So I see a subtle difference here. I completely endorse what's being said. The rules of connectome
*  interpretation, whatever they are, are not going to be the same in every brain region. The rules
*  are going to be different and the rules are going to be based on the innate information that's there
*  that are built for the affordances in the world that that brain region evolved to operate in.
*  And maybe there's nested hierarchies that are based on a fundamental connectome interpretation
*  logic. But this is part of the problem. Certainly the problem for those of us who work with rodents,
*  and I think it's one of the reasons why working with C. elegans and other organisms of that size
*  is so important. Okay, thank you for the questions from the audience. Thank you to
*  Aspirational Neuroscience for putting this together and to the panelists for a great discussion. Thanks, everyone.
*  Thank you.
*  I alone produce brain inspired. If you value this podcast, consider supporting it through Patreon
*  to access full versions of all the episodes and to join our discord community. Or if you want to
*  learn more about the intersection of neuroscience and AI, consider signing up for my online course
*  Neuro AI the quest to explain intelligence go to brand inspired co to learn more to get in touch with me email Paul at
*  Brand inspired co you're hearing music by the new year find them at the new year net. Thank you
*  Thank you for your support. See you next time
*  You
